# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/xkeyboard-config/hotfix-ln-sf.patch
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

ln: failed to create symbolic link '/usr/X11R7/share/X11/xkb/rules/xorg': File exists

FAILED: install script '/bin/sh -c ln -s base $DESTDIR/usr/X11R7/share/X11/xkb/rules/xorg' exit code 1, stopped

--- xkeyboard-config-2.34/rules/meson.build.vanilla	2022-04-12 11:44:38.084575836 +0200
+++ xkeyboard-config-2.34/rules/meson.build	2022-04-12 11:45:25.839573931 +0200
@@ -169,7 +169,7 @@
 if get_option('xorg-rules-symlinks')
 	foreach suffix: ['', '.lst', '.xml']
 		meson.add_install_script('sh', '-c',
-			'ln -s base@0@ $DESTDIR@1@/xorg@0@'.format(suffix, dir_xkb_rules))
+			'ln -sf base@0@ $DESTDIR@1@/xorg@0@'.format(suffix, dir_xkb_rules))
 	endforeach
 endif
 # Copy the DTD to the build directory, the man page generation expects it in
